src/components/TabPanel.js

```diff
@@ -24,28 +25,37 @@ module.exports = React.createClass({
   getDefaultProps() {
     return {
       selected: false,
+      selectedClassName: null,
+      style: {},
       id: null,
       tabId: null,
     };
   },
 
   render() {
-    const { className, children, selected, id, tabId, style, ...attributes } = this.props;
+    const {
+      id, tabId, selected,
+      className, selectedClassName, style, children,
+      ...attrs,
+    } = this.props;
+
+    // Only hide using inline styles if `selectedClassName` isn't provided
```

Comment:
The idea is that if you specify your own custom classes, you want control over hiding/showing. With inline styles you would have to override using `!important`.